velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1041,4 +1041,30 @@ struct ParseDateTimeFunction {
   }
 };
 
+template <typename T>
+struct DateFunction : public TimestampWithTimezoneSupport<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<Date>& result,
+      const arg_type<Timestamp>& input) {
+    static const int32_t kSecsPerDay{86'400};
```

Comment:
There is a static constant kSecsPerDay in TimestampConversion.h. Please use that in this code if possible.

Reply:
Done. Probably something that could be changed in the `cast` code as well.